### DISCUSSION TITLE
添加下载地图功能

### DISCUSSION THREAD

**Discussion by RainbowTaro2022** [Meloong-Git/PCL#2811] [提问] [answered]
HMCL里已经可以下载地图了
那如果PCL里也可以下载地图就好了

**Comment by WForst-Breeze** [comment, marked as answer]
之前说由于语言原因不会制作

**Reply by RainbowTaro2022** [reply]
先将地图发给翻译/汉化团队进行汉化，然后在制作

**Reply by WForst-Breeze** [reply]
不太现实，因为这样做就完全没有稳定的途径获取地图，并且让汉化组把所有地图都翻译一遍也不现实（）百公里油耗一个汉化组，这样的工作量可想而知

**Reply by MoYuan-CN** [reply]
地图中需要翻译的东西可能有很多，如果都要汉化的话翻译组不得忙死（）
